### DISCUSSION TITLE
【常见问题】为什么 Minihud 显示帧数异常？

### DISCUSSION THREAD

**Discussion by ScandiumMascot** [Scandium-Studio/Frodium-Modpack#21] [Q&A 你问我答] [answered]
当开启 Minihud Main Rendering 后，发现左上角显示帧数始终为 10 fps，而 Sodium 内置的帧数显示则可以正常显示帧数。

**Comment by WForst-Breeze** [comment, marked as answer]
由于 Exordium mod 的已知漏洞，会致使 Minihud 无法正常显示帧数，但并不影响正常游戏。如您需查看帧数，可以使用 F3 或 Sodium
